hooks/useGovernanceAssets.ts

```diff
@@ -162,6 +164,41 @@ export default function useGovernanceAssets() {
         canUseTokenTransferInstruction &&
         realm?.account.config.useCommunityVoterWeightAddin,
     },
+    {
+      id: Instructions.CreateAssociatedTokenAccount,
+      name: 'Create Associated Token Account',
+      isVisible: canUseAnyInstruction,
+    },
+    {
+      id: Instructions.CreateSolendObligationAccount,
+      name: 'Solend: Create Obligation Account',
+      isVisible: canUseAnyInstruction,
+    },
```

Comment:
please use correct visibility helpers, i assume that some of the following instructions should only be visible if users have access to token transfers etc.